src/features/sc-peripherals.md

```diff
@@ -82,7 +82,7 @@ There is an online .3dm to .3dj converter here: <https://3dj.lem.sh/>
 
 #### Example
 
-All arguments except for `shapesOff`, `shapesOn`, `bounds` and `texture` are optional.
+All arguments except for `shapesOff`, `shapesOn`, `bounds` are optional.
```

Comment:
`shapesOn` and `bounds` would be more gramatically correct whether or not you decide to add the oxford comma is up to you